src/Abstractions/NexusMods.Abstractions.Games/SortableItemViewModel.cs

```diff
@@ -0,0 +1,45 @@
+using System.Reactive;
+using NexusMods.Abstractions.UI;
+using ReactiveUI;
+
+namespace NexusMods.Abstractions.Games;
+
+public class SortableItemViewModel : AViewModel< ISortableItemViewModel>, ISortableItemViewModel
+{
+    public SortableItemViewModel(ISortableItem sortableItem)
+    {
+        SortableItem = sortableItem;
+        SortIndex = sortableItem.SortIndex;
+        DisplayName = sortableItem.DisplayName;
+        GroupName = SortableItem.ModName;
+        
+        MoveUp = ReactiveCommand.CreateFromTask(async () =>
+            {
+                await sortableItem.SortableItemProvider.SetRelativePosition(SortableItem, delta: 1);
+                return Unit.Default;
+            }
+        );
+        
+        MoveDown = ReactiveCommand.CreateFromTask(async () =>
+            {
+                await sortableItem.SortableItemProvider.SetRelativePosition(SortableItem, delta: -1);
+                return Unit.Default;
+            }
+        );
+        
+        MoveTo = ReactiveCommand.CreateFromTask<int, Unit>(async index =>
+            {
+                await sortableItem.SortableItemProvider.SetRelativePosition(SortableItem, index - SortIndex);
+                return Unit.Default;
+            }
+        );
+    }
+
+    public ISortableItem SortableItem { get; }
```

Comment:
Small style note. We usually put the properties before the constructors in this codebase.
No special specific reason, but figured I'd mention it.

Reply:
👍, I'll probably update that in next PR